c/src/ml-api-common-tizen-feature-check.c

```diff
@@ -97,46 +100,50 @@ _ml_tizen_get_feature_enabled (ml_feature_e ml_feature)
   ml_tizen_initialize_feature_state ();
 
   g_mutex_lock (&feature_info->mutex);
-  feature_enabled = feature_info->feature_state;
+  feature_enabled = feature_info->feature_state[ml_feature];
   g_mutex_unlock (&feature_info->mutex);
 
   if (NOT_SUPPORTED == feature_enabled) {
-    _ml_loge ("machine_learning.inference NOT supported");
+    _ml_loge ("Tizen machine learning feature (%s) is NOT supported.",
```

Comment:
Please add what the developer should do with this error.

E.g.,
Tizen machine learning feature (%s) is NOT supported. Please check if your application has properly requested the feature and the device has the feature installed.